modular_skyrat/modules/families/code/secret_handshake.dm

```diff
@@ -0,0 +1,67 @@
+/// Gangster secret handshakes.
+/obj/item/hand_item/slapper/secret_handshake
+	name = "Secret Handshake"
+	icon_state = "recruit"
+	icon = 'icons/obj/gang/actions.dmi'
+	/// References the active families gamemode handler (if one exists), for adding new family members to.
+	var/datum/gang_handler/handler
+	/// The typepath of the gang antagonist datum that the person who uses the package should have added to them -- remember that the distinction between e.g. Ballas and Grove Street is on the antag datum level, not the team datum level.
+	var/gang_to_use
+	/// The team datum that the person who uses this package should be added to.
+	var/datum/team/gang/team_to_use
+
+
+/// Adds the user to the family that this package corresponds to, dispenses the free_clothes of that family, and adds them to the handler if it exists.
+/obj/item/hand_item/slapper/secret_handshake/proc/add_to_gang(mob/living/user, original_name)
+	var/datum/antagonist/gang/swappin_sides = new gang_to_use()
+	swappin_sides.original_name = original_name
+	swappin_sides.handler = handler
+	user.mind.add_antag_datum(swappin_sides, team_to_use)
+	var/policy = get_policy(ROLE_FAMILIES)
+	if(policy)
+		to_chat(user, policy)
+	team_to_use.add_member(user.mind)
+	swappin_sides.equip_gangster_in_inventory()
+	if (!isnull(handler) && !handler.gangbangers.Find(user.mind)) // if we have a handler and they're not tracked by it
+		handler.gangbangers += user.mind
```

Comment:
Space out your code, I'm not going to comment it everywhere but please, it's taking so much longer to review this because it's so dense, you have no idea.

Reply:
Just space out the whole file. And all the other ones, really.